test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/TagHelpersWebSite.Home.About.html

```diff
@@ -12,8 +12,8 @@
 <body>
     <h1 style="font-family: cursive;">ASP.NET vNext - About</h1>
     <p>| <a href="/" style="background-color: gray;color: white;border-radius: 3px;border: 1px solid black;padding: 3px;font-family: cursive;">My Home</a>
-       | <a href="/home/about" style="background-color: gray;color: white;border-radius: 3px;border: 1px solid black;padding: 3px;font-family: cursive;">My About</a> 
-       | <a href="/home/help" style="background-color: gray;color: white;border-radius: 3px;border: 1px solid black;padding: 3px;font-family: cursive;">My Help</a> |</p>
+       | <a href="/Home/About" style="background-color: gray;color: white;border-radius: 3px;border: 1px solid black;padding: 3px;font-family: cursive;">My About</a> 
```

Comment:
For a route template like `{controller}/{action}` the casing of these would the case that is passed at the call site, but in case of Dispatching, we use the values that are present on the tree (which is in turn the values off of action descriptors). Yeah there is a change here, but it should not be of a concern as these links can still be reached. Also there's a workaround where users can lowercase urls if they want to